README.md

```diff
@@ -1,4 +1,38 @@
-# React-Play Development Kit
+# Create React Play
+
+## Welcome to  `reactplay.io` ecosystem
```

Comment:
Have we changed the react-play repo read me accordingly? I guess that project will not have any steps for the create play as for devs this will be the entry point?

Reply:
right